packages/dataviews/src/dataviews-layouts/index.ts

```diff
@@ -19,21 +19,22 @@ import { LAYOUT_GRID, LAYOUT_LIST, LAYOUT_TABLE } from '../constants';
 import type { View, Field } from '../types';
 import PreviewSizePicker from './grid/preview-size-picker';
 import DensityPicker from './table/density-picker';
+import SortHierarchical from './table/sort-hierarchical';
 
 export const VIEW_LAYOUTS = [
 	{
 		type: LAYOUT_TABLE,
 		label: __( 'Table' ),
 		component: ViewTable,
 		icon: blockTable,
-		viewConfigOptions: DensityPicker,
+		viewConfigOptions: [ SortHierarchical, DensityPicker ],
```

Comment:
nit: by leaving as it was before and adding `<>`, I think it's more flexible as the layout could also add some separators or wrappers.